Makefile

```diff
@@ -662,61 +794,46 @@ install.libseccomp.sudo:
-	for shell in $${!outfiles[*]}; do \
-	    for remote in "" "-remote"; do \
-	        podman="podman$$remote"; \
-	        outfile=$$(printf "completions/$$shell/$${outfiles[$$shell]}" $$podman); \
-	        ./bin/$$podman completion $$shell >| $$outfile; \
-	    done;\
+.PHONY: uninstall
+uninstall:
+	for i in $(filter %.1,$(MANPAGES_DEST)); do \
+		rm -f $(DESTDIR)$(MANDIR)/man1/$$(basename $${i}); \
+	done; \
+	for i in $(filter %.5,$(MANPAGES_DEST)); do \
+		rm -f $(DESTDIR)$(MANDIR)/man5/$$(basename $${i}); \
 	done
+	# Remove podman and remote bin
+	rm -f $(DESTDIR)$(BINDIR)/podman
+	rm -f $(DESTDIR)$(BINDIR)/podman-remote
+	# Remove related config files
+	rm -f ${DESTDIR}${ETCDIR}/cni/net.d/87-podman-bridge.conflist
+	rm -f ${DESTDIR}${TMPFILESDIR}/podman.conf
+	rm -f ${DESTDIR}${SYSTEMDDIR}/io.podman.socket
+	rm -f ${DESTDIR}${USERSYSTEMDDIR}/io.podman.socket
+	rm -f ${DESTDIR}${SYSTEMDDIR}/io.podman.service
+	rm -f ${DESTDIR}${SYSTEMDDIR}/podman.service
+	rm -f ${DESTDIR}${SYSTEMDDIR}/podman.socket
+	rm -f ${DESTDIR}${USERSYSTEMDDIR}/podman.socket
+	rm -f ${DESTDIR}${USERSYSTEMDDIR}/podman.service
 
-.PHONY: validate.completions
-validate.completions: SHELL:=/usr/bin/env bash # Set shell to bash for this target
-validate.completions:
-	# Check if the files can be loaded by the shell
-	. completions/bash/podman
-	if [ -x /bin/zsh ]; then /bin/zsh completions/zsh/_podman; fi
-	if [ -x /bin/fish ]; then /bin/fish completions/fish/podman.fish; fi
-
-.PHONY: validate
-validate: gofmt lint .gitvalidation validate.completions man-page-check swagger-check tests-included
-
-.PHONY: build-all-new-commits
-build-all-new-commits:
-	# Validate that all the commits build on top of $(GIT_BASE_BRANCH)
-	git rebase $(GIT_BASE_BRANCH) -x make
-
-.PHONY: build-no-cgo
-build-no-cgo:
-	env BUILDTAGS="containers_image_openpgp exclude_graphdriver_btrfs exclude_graphdriver_devicemapper exclude_disk_quota" CGO_ENABLED=0 $(MAKE)
-
-.PHONY: vendor
-vendor:
-	GO111MODULE=on $(GO) mod tidy
-	GO111MODULE=on $(GO) mod vendor
-	GO111MODULE=on $(GO) mod verify
-
-.PHONY: vendor-in-container
-vendor-in-container:
-	podman run --privileged --rm --env HOME=/root -v `pwd`:/src -w /src docker.io/library/golang:1.16 make vendor
-
-.PHONY: package
-package:  ## Build rpm packages
-	## TODO(ssbarnea): make version number predictable, it should not change
-	## on each execution, producing duplicates.
-	rm -rf build/* *.src.rpm ~/rpmbuild/RPMS/*
-	./contrib/build_rpm.sh
-
-# Remember that rpms install exec to /usr/bin/podman while a `make install`
-# installs them to /usr/local/bin/podman which is likely before. Always use
-# a full path to test installed podman or you risk to call another executable.
-.PHONY: package-install
-package-install: package  ## Install rpm packages
-	sudo ${PKG_MANAGER} -y install ${HOME}/rpmbuild/RPMS/*/*.rpm
-	/usr/bin/podman version
-	/usr/bin/podman info  # will catch a broken conmon
+.PHONY: clean
+clean: ## Clean all make artifacts
+	rm -rf \
```

Comment:
Do you really want -f? Could ignoring rm errors cause any unexpected results?

Reply:
That wasn't a change I introduced...just looked, this is on master as well.  I can change it if there's good reason, but it wasn't really in-scope for my recipe adjustments (mainly targeted at the `podman-*-release` targets and getting non-linux binaries into their own subdir).

Reply:
Please leave it. `rm -f` and `-rf` has been the Makefile convention for as long as I can remember. There is good reason for it.

Reply:
Ed's explanation (below) also gives me confidence we should leave this as-is.